### PR TITLE
[NodeBundle] View parent nodes from list

### PR DESCRIPTION
**Pull request by dannyvw**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | comma separated list of tickets fixed by the PR

### CODE REVIEW

File: src/Kunstmaan/NodeBundle/Controller/NodeAdminController.php [hunk 1]
```diff
@@ -126,26 +126,7 @@ public function indexAction(Request $request): Response
...
-        $acl = $this->authorizationChecker;
-        $itemRoute = function (EntityInterface $item) use ($locale, $acl) {
-            if ($acl->isGranted(PermissionMap::PERMISSION_VIEW, $item->getNode())) {
-                return [
-                    'path' => '_slug_preview',
-                    'params' => ['_locale' => $locale, 'url' => $item->getUrl()],
-                ];
-            }
-        };
-        $nodeAdminListConfigurator->addSimpleItemAction('action.preview', $itemRoute, 'eye');
```
Comment (acrobat) [inline src/Kunstmaan/NodeBundle/Controller/NodeAdminController.php:147]: This code is not added in `getAdminListConfigurator` but is still needed I think?

File: src/Kunstmaan/NodeBundle/AdminList/NodeAdminListConfigurator.php [hunk 2]
```diff
@@ -46,6 +48,8 @@ class NodeAdminListConfigurator extends AbstractDoctrineORMAdminListConfigurator
      */
     protected $authorizationChecker;
 
+    protected ?Node $node = null;
```
Comment (acrobat) [inline src/Kunstmaan/NodeBundle/AdminList/NodeAdminListConfigurator.php]: I would make this private for BC reasons, we can still expose it if needed in the future

**Review by acrobat — CHANGES_REQUESTED** [review]
Can you also rebase against the latest 6.x branch, this should fix the failing tests? Thanks!

File: src/Kunstmaan/NodeBundle/Controller/NodeAdminController.php (hunk 1 above)
Comment (dannyvw) [reply src/Kunstmaan/NodeBundle/Controller/NodeAdminController.php:147]: See https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/3334/files#diff-ec9c9d7d2b1ae0e92e0c5bee37ebb65e41a8abcd9a95c9eae80665cf998c30f7R132 or do you mean something else?

Comment (acrobat) [reply src/Kunstmaan/NodeBundle/Controller/NodeAdminController.php:147]: Oh I have missed it somehow, all good!

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thanks @dannyvw

**Pull request merged**